health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -205,11 +211,37 @@
+    return temp.toUpperCase().replace(/ /g, "_");
+  }
+
   const generateCodes = (questions) => {
     const codes = {};
     const local = [];
     let activeCounters = { top: 0 }; // Track active question counts at each level
 
+    // Precompute common values once
+    let checklistTypeTemp = LocalisationCodeUpdate(checklistType);
+    if (checklistTypeCode) checklistTypeTemp = checklistTypeCode;
+    let roleTemp =  LocalisationCodeUpdate(role);
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'roleTemp' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)